src/Queue.ts

```diff
@@ -27,7 +27,7 @@ export class Queue {
   }
 
   public topic(topic: string) {
```

Comment:
public topic(...topics: string) { 
    this.options.topics.push(...topics);
}

mais detalhes em um proximo comentário =)